.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: ":bug: Bug Report"
+about: Create an issue related to a bug or encountered problem.
+title: ":bug: [BUG]: "
+labels: bug
+assignees: tlvu, fmigneault
```

Comment:
@fmigneault you want to be the CRIM's default assignee? In the contribution, we proposed @MatProv .

Reply:
I can swap it. Good with you? @MatProv

Reply:
Yes, haven't notice this in the template but makes more sense in fact. Please change it.